tests/bug-7241-02-pre8/test.rules

```diff
@@ -0,0 +1 @@
+drop tcp any any -> any any (flow:established; app-layer-protocol:!tls; sid:1;)
```

Comment:
nit : could add a README.md, especially for explaining why we have 2 different tests for 7 and 8